http/get_simple/ruby/client/client.rb

```diff
@@ -0,0 +1,33 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+require "net/http"
+require "arrow"
+
+uri = URI("http://localhost:8008")
+
+start = Time.now
+arrows_data = Net::HTTP.get(uri)
+input = Arrow::BufferInputStream.new(arrows_data)
+reader = Arrow::RecordBatchStreamReader.new(input)
+schema = reader.schema
+table = reader.read_all
```

Comment:
Ah, I forgot that we can use `Arrow::Table.load` for this:

```ruby
arrows_data = Net::HTTP.get(uri).freeze
table = Arrow::Table.load(Arrow::Buffer.new(arrows_data), format: :arrows)
```

Reply:
Thanks! I opened #22 to simplify it like this.